contents/handbook/strategy/roadmap.md

```diff
@@ -4,66 +4,48 @@ sidebar: Handbook
 showTitle: true
 ---
 
-## Up-to-date roadmap
+Our mission is to make more successful products in the world. 
 
-We keep our most up-to-date roadmap on [GitHub](https://github.com/orgs/PostHog/projects/1).
+Our roadmap for 2021 will do three things:
+1. Create a solid core product that's easy to use
+2. Have the best developer platform for event-based analytics
+3. Set us up to service huge volumes
 
-## General Product Strategy
+# 1. Core product
 
-### Open Core Business Model
+PostHog is a product that people love, primarily because covers 90% of analytics use cases but bundled into one package.
+Some examples of the functionality we've built last year:
 
-We aim for every company that builds software to be using PostHog.
+- Product analytics
+- Session recording
+- Feature flags
+- Heatmaps
+- Autocapture
 
-* The open source platform is designed to be installed by a developer when they start a new project, before a Product Team is in place. There is no competition at this stage.
-* The open source platform is also well suited to adoption in an existing startup, midmarket or enterprise customer as it has no information security and no vendor risk management requirements. We are much lower friction.
-* Providing self-hosted capabilities means we have complete flexibility over how we charge customers. This means we can align with higher volume customers.
+There's plenty of work to be done within those categories to make a product that is especially useful for engineers and other product minded people. To add on to that, this year we want to build:
 
-### Becoming a Piece of Infrastructure
+- A/B testing
+- User feedback
+- Data pipelines
 
-It is key that we don't see companies grow out of PostHog over time.
+On top of those new categories we have a lot of work to do to make our product more stable at higher volumes (especially when self-deploying), much easier to get started with and to catch up with other state-of-the-art analytics software.
 
-The more PostHog can become part of our customers' infrastructure, the better.
+# 2. Best developer platform
 
-The ways to achieve this are (i) by becoming a data store (ii) by ensuring extensibility through integrating with other tools and (iii) by the breadth of users powered by the tool in the clients that adopt us.
+Developers like using PostHog for many reasons. We're open-source at our core, which has helped a huge amount in gaining trust and adoption from the developer community.
+It's easy to debug, you can self-host and PostHog is now starting to become extensible.
 
+This year we're going to lean into that last item. We've kept plugins relatively quiet so far, but we believe plugins will be what will make PostHog the default choice for developers.
```

Comment:
The way open source makes it super easy to become a platform is much greater than the marketing benefit, even though we have felt the latter make such a huge impact so far.